Week-11/lightswitch/src/action/action.js

```diff
@@ -0,0 +1,5 @@
+export const changeLight = (value) => {
+  return (dispatch) => {
+    dispatch({ type: "change", payload: value });

```

Comment:
Type value will be in all capital letters like CHANGE_THEME. It's naming convention.